tailwind.config.ts

```diff
@@ -20,6 +20,7 @@ const config: Config = {
       },
       screens: {
         smr: '480px',
+        strange: '1169px',
```

Comment:
I'm guessing this is for the weird screen size between `lg` and `xl`? Not sure if I like the naming though, as it's a bit ambiguous without comments or out of context. I'd consider renaming it for better clarity

Reply:
Yes, and I just renamed it with extra comments, thanks for pointing out